include/seqan3/alphabet/structure/structured_rna.hpp

```diff
@@ -97,7 +97,7 @@ class structured_rna :
     /*!\name Constructors, destructor and assignment
      * \{
      */
-    constexpr structured_rna() : base_type{} {}
+    constexpr structured_rna() noexcept : base_type{} {}
     constexpr structured_rna(structured_rna const &) = default;
     constexpr structured_rna(structured_rna &&) = default;
     constexpr structured_rna & operator =(structured_rna const &) = default;
```

Comment:
I cannot mark the proper line as it is not changed but you maybe want to add the noexcept specifier to the doxygen documentation in line 110? (copied docs from cartesian composition)